common/netlink.cpp

```diff
@@ -0,0 +1,468 @@
+#include "netlink.h"
+
+#include <linux/if_arp.h> // ARPHRD_ defines
+
+#include <unistd.h> // getpid(), getpagesize()
+#include <string.h> // memcpy, memset etc
+
+Netlink::Netlink()
```

Comment:
Maybe NetlinkManager? But on the other hand, the underlying implementation should not be that interesting to whoever uses this, as long as one can use them both inside and outside containers.

NetworkManager could work, but it has a very GNOMEish ring to it ;)